ractor/Cargo.toml

```diff
@@ -19,19 +19,20 @@ cluster = []
 tokio_runtime = ["tokio/time", "tokio/rt", "tokio/macros", "tokio/tracing"]
 
 # default = ["async-std"]
-default = ["tokio_runtime"]
+default = ["tokio_runtime", "async-trait"]
```

Comment:
I think that should be

``` suggestion
default = ["tokio_runtime", "dep:async-trait"]
```

to make that optional ([see here](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies))